capstone/src/main/java/com/google/univiz/api/SearchData.java

```diff
@@ -1 +1,3 @@
 package com.google.univiz.api;
+
+import java.util.List;
```

Comment:
Possibly add the java doc for the SearchData interface or otherwise describe it in the PR.

Reply:
took these changes away from searchdata :)